packages/toolpad-core/src/Account/Account.tsx

```diff
@@ -1,48 +1,37 @@
 import * as React from 'react';
-import { styled } from '@mui/material/styles';
 import PropTypes from 'prop-types';
+import Button from '@mui/material/Button';
 import Popover from '@mui/material/Popover';
 import Divider from '@mui/material/Divider';
-import Button, { ButtonProps } from '@mui/material/Button';
-import IconButton, { IconButtonProps } from '@mui/material/IconButton';
-import Tooltip from '@mui/material/Tooltip';
-import Logout from '@mui/icons-material/Logout';
-import { Typography } from '@mui/material';
-import { SessionAvatar } from './SessionAvatar';
+import { SignInButton } from './SignInButton';
+import { SignOutButton } from './SignOutButton';
+import { AccountPreview, AccountPreviewProps } from './AccountPreview';
+import { AccountPopoverHeader } from './AccountPopoverHeader';
+import { AccountPopoverFooter } from './AccountPopoverFooter';
 import { SessionContext, AuthenticationContext } from '../AppProvider/AppProvider';
-import DEFAULT_LOCALE_TEXT from '../shared/locales/en';
-
-const AccountInfoContainer = styled('div')(({ theme }) => ({
-  display: 'flex',
-  flexDirection: 'row',
-  justifyContent: 'space-between',
-  padding: theme.spacing(2),
-  gap: theme.spacing(2),
-}));
-
-const SignOutContainer = styled('div')(({ theme }) => ({
-  display: 'flex',
-  flexDirection: 'row',
-  padding: theme.spacing(1),
-  justifyContent: 'flex-end',
-}));
+import { LocaleProvider, useLocaleText } from '../shared/locales/LocaleContext';
 
 export interface AccountSlots {
+  /**
+   * The component used for the account preview
+   * @default AccountPreview
+   */
+  preview?: React.ElementType;
+  /**
+   * The component used for the content of account popover
+   * @default Popover
+   */
+  popoverContent?: React.ElementType;
```

Comment:
I think there's a naming mismatch between some properties in `slots` and `slotProps` right now. Shouldn't they always exactly match?

Or I might be wrong, maybe it was just something I assumed in the other components I've worked on... It feels more intuitive that they match...

Reply:
Agreed, it should have in this case as well but I missed one `slot` and one `slotProp` which is causing confusion here; addressed that